tachyon/math/polynomials/univariate/univariate_evaluation_domain.h

```diff
@@ -455,6 +474,7 @@ class UnivariateEvaluationDomain : public EvaluationDomain<F, MaxDegree> {
     size_t size = poly_or_evals.NumElements();
     size_t num_elems_per_thread = std::max(size / thread_nums, size_t{1024});
     OPENMP_PARALLEL_FOR(size_t i = 0; i < size; i += num_elems_per_thread) {
+      TRACE_EVENT("EvaluationDomain", "MultiplyLoop");
```

Comment:
This is not descriptive. 

```suggestion
      TRACE_EVENT("EvaluationDomain", "UnivariateEvaluationDomain::DistributePowersAndMulByConst::ParallelLoop");
```

Reply:
I think this should be labeled `Subtask`. And since it will be already inside the `UnivariateEvaluationDomain::DistributePowersAndMulByConst` slice, we wont need to be that descriptive I think.

Reply:
You said it should be labeled `Subtask`, but you didn't